### PR TITLE
Add eslint and action

### PR DESCRIPTION
**Pull request by peterthehan**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "commonjs": true,
+    "es2020": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 2018
```
Comment (peterthehan) [inline .eslintrc.json:9]: https://eslint.org/docs/user-guide/getting-started#installation-and-usage

running `npx eslint --init` initialized the `ecmaVersion` to `12` but my vscode was showing some kind of warning, so i downgraded. dunno what to do about that fyi.

**Comment by Bytooo** [comment]
I would look into the Airbnb JS Styling Guide which is the most popular set of JS Styling Rules

- **Guide**: https://github.com/airbnb/javascript

- **eslint config**: https://www.npmjs.com/package/eslint-config-airbnb-base

**Comment by peterthehan** [comment]
https://github.com/peterthehan/create-discord-bot/pull/45#issuecomment-678883901
are js styling rules/guides necessary if we're relying on prettier?

**Comment by Bytooo** [comment]
Its not necessarily styling rules there are also best code practices and other stuff that Prettier doesn't include, having both ensures that our code is as clean and professional as possible in my opinion.

**Pull request merged**